doc/builders/trivial-builders.chapter.md

```diff
@@ -1,14 +1,29 @@
-# Trivial builders {#chap-trivial-builders}
+# Trivial Builders {#chap-trivial-builders}
 
-Nixpkgs provides a couple of functions that help with building derivations. The most important one, `stdenv.mkDerivation`, has already been documented above. The following functions wrap `stdenv.mkDerivation`, making it easier to use in certain cases.
+Nixpkgs provides a variety of functions that help with building derivations.
+The most important of these, `stdenv.mkDerivation`, has already been documented above.
+The following functions wrap `stdenv.mkDerivation`, making it easier to use in certain cases.
 
 ## `runCommand` {#trivial-builder-runCommand}
 
-This takes three arguments, `name`, `env`, and `buildCommand`. `name` is just the name that Nix will append to the store path in the same way that `stdenv.mkDerivation` uses its `name` attribute. `env` is an attribute set specifying environment variables that will be set for this derivation. These attributes are then passed to the wrapped `stdenv.mkDerivation`. `buildCommand` specifies the commands that will be run to create this derivation. Note that you will need to create `$out` for Nix to register the command as successful.
+This takes three arguments, `name`, `env`, and `buildCommand`:
 
-An example of using `runCommand` is provided below.
+* `name` is just the name that Nix will append to the store path in the same way that `stdenv.mkDerivation` uses its
+  `name` attribute.
+* `env` is an attribute set, specifying environment variables that will be available in the build script for this derivation.
+  These attributes are passed through in the underlying call to `stdenv.mkDerivation`.
+* `buildCommand` is a `bash` script that will be run to build this derivation.
+  Note that you will need to create `$out` as part of the build script for Nix to consider the command as having executed successfully.
```

Comment:
Point out `$out` will be an environment variable available in the build script pointing to the resulting nix store path.

Maybe link to `stdenv.mkDerivation` multiple outputs, default output is `out`, etc.